### ISSUE TITLE
HTTP ERROR 500 installed everything

### ISSUE THREAD

**Issue by webdsf** [WWBN/AVideo#7181]
Hello, I followed the installation instructions and did not understand what was happening, the main site does not open, I installed an encoder on this server, everything works and starts up.
![111](https://user-images.githubusercontent.com/60380221/192339196-92e952e5-2f7e-4ff9-9bfe-5982b9d4238c.jpg)

**Comment by DanielnetoDotCom** [comment]
not sure, what is your problem, but seems to be a file permission issue.

to be able to resolve that you may need some Linux background knowledge

**Comment by webdsf** [comment]
> not sure, what is your problem, but seems to be a file permission issue.

![Screenshot_3](https://user-images.githubusercontent.com/60380221/192347646-b85869ca-a59f-4a16-9c1e-10f60640e10a.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/60380221/192347651-f4368b69-fb77-443c-8de2-ac5e945197cd.jpg)
![Screenshot_1](https://user-images.githubusercontent.com/60380221/192347656-91d6c667-8ab8-41f9-88dc-bfa0761a46f5.jpg)
In the latter, I tried to change to this, but it didn't work. There was a root. It seems to have installed it before and everything worked.

**Comment by DanielnetoDotCom** [comment]
you are looking at the wrong files. 

it should be videos/configuration.php

**Comment by webdsf** [comment]
> you are looking at the wrong files.

I did not get to the installation and I do not have such a folder and file.

**Comment by DanielnetoDotCom** [comment]
why not?

if you use git pull you should have all. 

if you did not use git pull that explains the issue.

**Comment by lunar54** [comment]
I also had 500 issue running a new setup. if you do domain/install install pages shows but if you just go to domain it does not re direct to install page. encoder setup seems to be fine . did see an error message about document type not being defined in your index.php. I had also used git and did not see a config page either and videos directory in AVideo/AVideo-Encoder is made when your on install page. if there suppose to be created during your git pull they are not being created I have always had to create them.

**Comment by DanielnetoDotCom** [comment]
you may need help to install AVideo. 

I cannot teach you how to handle linux.

usually if you have the correct server with a clean Ubuntu you do not have problems

**Comment by lunar54** [comment]
what do you mean by correct server ? and i have done AVideo installs before with no issues . but would say your new install doc for 22.04 you explain how to setup apache virtual host but you dont say how to setup nginx server block.
https://github.com/WWBN/AVideo/issues/6555#issuecomment-1185143589

**Comment by DanielnetoDotCom** [comment]
It is clear you are missing something, but I do not know what is it

also based on your screenshots Incan see you do not have a basic installation

**Comment by lunar54** [comment]
no because i know how to setup virtual host and ngnix blocks and also run nginx at start up with nginx.service.
How ever something in your files that have changed or just maybe something flaky in new installs with Ubuntu 22.04 is giving issue gonna try Deb 11

**Comment by lunar54** [comment]
Deb 11 does the same  you can reach the install page by domain.com/install but domain.com errors out and does not matter if root or www-data owns them same results.

**Comment by DanielnetoDotCom** [comment]
you have never answered 

Did you use `git pull`?

sorry but based on your comment here https://github.com/WWBN/AVideo/issues/7181#issuecomment-1258554860 I suspect you are not familiar with git and you need to find someone to do it for you.

**Comment by lunar54** [comment]
your install docs say to use  git clone https://github.com/WWBN/AVideo.git ?

**Comment by DanielnetoDotCom** [comment]
If you use git clone you still missing something. sorry I have no idea what you are missing.

**Comment by lunar54** [comment]
So then why does your docs say to use it ?
https://imgur.com/a/eV9AFvg

**Comment by DanielnetoDotCom** [comment]
you are right, I mean git clone. and I use the same command in all my installations. and I never had any problem.

also, I saw you are having problems with the encoder. did you install the encoder?

**Comment by lunar54** [comment]
when you go to domain.com it will not go to domain.com/install as it should but add install to domain.com ( domain.com/install ) install page shows up. Encoder install page shows up Have tried to install once but after removing install dir same nothing at domain.com

**Comment by DanielnetoDotCom** [comment]
you should not remove the install dir.

I will remove this instructions later

**Comment by DanielnetoDotCom** [comment]
I found something I would like you to run a git pull and try again soon

**Comment by DanielnetoDotCom** [comment]
Ok, please update and try again. I hope it is fixed now.

**Comment by lunar54** [comment]
I ran new install on Ubuntu 22.04 main page shows up as well as encoder page and installs.
I did notice with new install doc for 22.04 seems to pull the wrong nginx config file. You still have to edit anyways to change domain in it still correct ? Also looking at your Health checker my server sitting on a 128gb disk using 3,4 gb but checker is saying i have less than 49gb space left in video dir but dir 180K in size. How is Health checker calculating size left ?
I have not gone into ssl stuff as of yet.
